### PR TITLE
43 add navbar

### PR DESCRIPTION
**Pull request by danielKNS**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -2,6 +2,17 @@
...
+   align-items: center;
+   justify-content: space-evenly;
+   width: 100%;
+   height: 50px;
+   padding: 30px;
+
+}
 
 
   .button{
```
Comment (aMotherFox) [inline src/App.css]: I think .button can be shift + tabbed

File: src/App.tsx [hunk 2]
```diff
@@ -1,38 +1,26 @@
...
-
+       <BrowserRouter>
+            <NavBar />
+                <Routes>
+                    <Route path="/" element={<Home/>}/>
+                    <Route path="/entrees" element={<Entrees/>}/>
+                     <Route path="/appetizers" element={<Appetizers/>}/>
+                </Routes>
+       </BrowserRouter>
+        <p>Working on the Navbar! </p>
```
Comment (aMotherFox) [inline src/App.tsx:19]: The tabbing is off on line 18 and 21

File: src/App.css (hunk 1 above)
Comment (danielKNS) [reply src/App.css]: Sure

File: src/App.tsx (hunk 2 above)
Comment (danielKNS) [reply src/App.tsx:19]: Let me fix that

**Review by aMotherFox — DISMISSED** [review]

File: package.json [hunk 3]
```diff
@@ -14,7 +14,10 @@
     "axios": "^1.1.2",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-router": "^6.4.2",
+    "react-router-dom": "^6.4.2",
     "react-scripts": "5.0.1",
+    "styled-components": "^5.3.6",
```
Comment (LucasCostaAtCyberSaint) [inline package.json]: are we using this dependency for anything?

File: src/App.tsx [hunk 4]
```diff
@@ -1,38 +1,26 @@
...
-                Press to say Hello to the World
-            </button>
-        </div>
-
-        <div>
-           <p> Response Body from API: </p>
-            {finalResult.message}
-        </div>
-
+       <BrowserRouter>
```
Comment (LucasCostaAtCyberSaint) [inline src/App.tsx]: This should be on the root component

File: src/App.tsx [hunk 5]
```diff
@@ -1,38 +1,26 @@
...
+            <NavBar />
+                <Routes>
+                    <Route path="/" element={<Home/>}/>
+                    <Route path="/entrees" element={<Entrees/>}/>
+                    <Route path="/appetizers" element={<Appetizers/>}/>
+                </Routes>
+       </BrowserRouter>
+       <p>Working on the Navbar! </p>
```
Comment (LucasCostaAtCyberSaint) [inline src/App.tsx]: the tabbing here is incorrect

**Review by LucasCostaAtCyberSaint — APPROVED** [review]

**Review by aMotherFox — APPROVED** [review]

**Pull request merged**